server/services/Authentication/signup.go

```diff
@@ -0,0 +1,104 @@
+package authentication
+
+import (
+	"fmt"
+	"main/server/db"
+	"main/server/model"
+	"main/server/request"
+	"main/server/response"
+	"main/server/services/token"
+	"main/server/utils"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func AlreadyExists(data string) bool {
+
+	return db.RecordExist("users", data, "email")
+
+}
+
+func SignupService(ctx *gin.Context, input *request.AuthRequest) {
+
+	var user model.User
+
+	//check if the newPassword and password are the same
+	if input.NewPassword != input.Password {
```

Comment:
Why do we have two passwords at the time of Signup?

Reply:
i have removed one password  field ,eariler we thought the (password and re-enter password check) is done at backend so thats why there are 2 fields.